src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -0,0 +1,99 @@
+@import "scss/constants.scss";
+
+$grey--regular: #3f3d42;
+$grey--hover: #4c494f;
+$grey--users: #4c494f;
+$grey--ligth: #ebebeb;
+
+$purple--regular: #7c46fb;
+$purple--hover: #7c46fb;
+
+$ScheduleEvent--margin-top: 0.25rem;
+$ScheduleEvent--height: 3.75rem;
+$ScheduleEvent--padding: 0 0.25rem 0 0.75rem;
+$ScheduleEvent--border-radius: 18px;
+$ScheduleEvent--box-shadow: 0 5px 10px rgba(0, 0, 0, 0.65);
+
+$bookmark--padding: 0.5rem;
+$bookmark-hover--padding-top: 0.375rem;
+
+$description--margin-top: 2px;
+
+.ScheduleEvent {
+  display: flex;
+  position: absolute;
+  border-radius: $ScheduleEvent--border-radius;
+  padding: $ScheduleEvent--padding;
+  cursor: pointer;
+  height: $ScheduleEvent--height;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $grey--regular;
+  box-shadow: $ScheduleEvent--box-shadow;
+  margin-top: $ScheduleEvent--margin-top;
+  margin-left: var(--event--margin-left);
+  width: var(--event--width);
+
+  &:hover {
+    background-color: $grey--hover;
+    z-index: z(navbar__schedule-event--hover);
+  }
+
+  &--users {
+    color: $grey--hover;
+    background-color: $grey--ligth;
+
+    &:hover {
+      background-color: $white;
+    }
+
+    .ScheduleEvent__bookmark {
+      color: $black;
+    }
+  }
+
+  &--live {
+    background-color: $purple--regular;
+    color: $white;
+
+    &:hover {
+      background-color: $purple--hover;
+    }
+
+    .ScheduleEvent__bookmark {
+      color: $white;
+    }
+  }
+
+  &__info {
+    font-size: 0.8rem;
+    overflow: hidden;
+  }
+
+  &__title {
+    @include line-clamp-with-overflow(2);
+    white-space: pre-line;
+    font-weight: bold;
+  }
+
+  &__host {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: block;
+    margin-top: $description--margin-top;
+    font-size: 0.7rem;
```

Comment:
[NB] Same as above ^

Reply:
See my comments in https://github.com/sparkletown/sparkle/pull/1302#discussion_r633945091

Reply:
https://github.com/sparkletown/sparkle/pull/1302#discussion_r634302797

Reply:
not sure what to do with this one. 
It should be less than 12px (event title), and we have `$font-size--xs: 10px`, but IMO it's too small.

0.8 rem + 0.7 rem
![image](https://user-images.githubusercontent.com/63313732/118795460-99b7c800-b868-11eb-87c7-51a9f276f028.png)

12px + 10px (host label is more difficult to read)
![image](https://user-images.githubusercontent.com/63313732/118795565-b3f1a600-b868-11eb-9da5-66e51aea3259.png)

Should I create a new constant then (11px)? And how should I name it?
xs = 11px
xxs = 10px?

current list:
```scss
$font-size--xs: 10px;
$font-size--sm: 12px;
$font-size--md: 14px;
$font-size--lg: 16px;
$font-size--xl: 20px;
// Not sure this is the best naming for these
$font-size--xxl: 30px;
$font-size--xxxl: 50px;
```

Reply:
@mike-lvov Thoughts?